src/test/kotlin/uk/gov/dluhc/printapi/config/IntegrationTest.kt

```diff
@@ -112,6 +113,9 @@ internal abstract class IntegrationTest {
     @Autowired
     protected lateinit var certificateRepository: CertificateRepository
 
+    @Autowired
+    protected lateinit var temporaryCertificateRepository: TemporaryCertificateRepository
```

Comment:
Minor, @Neil-Massey ,
It will be good to clear this repository for each integration test as well, like the `CertificateRepository`.

Reply:
Good catch - added.